docs/API.md

```diff
@@ -1202,71 +1363,65 @@ enters a connected state.
-| ----------------- | ----- | ------- | --------- | ------ | -------- | ------ |
-| `SRTO_STATE`      |       | n/a     | `int32_t` |        | n/a      | n/a    |
+| OptName              | Since | Binding | Type       |  Units  |  Default  | Range  | Dir | Entity |
+| -------------------- | ----- | ------- | ---------- | ------- | --------- | ------ | --- | ------ |
+| `SRTO_STATE`         |       |         | `int32_t`  |  enum   |           |        | R   | S      |
+
+- Returns the current socket state, same as `srt_getsockstate`.
 
-- **[GET]** - UDT connection state. (See enum `SRT_SOCKSTATUS`)
 
 ---
 
-| OptName         | Since | Binding | Type          | Units  | Default  | Range      |
-| --------------- | ----- | ------- | ------------- | ------ | -------- | ---------- |
-| `SRTO_STREAMID` | 1.3.0 | pre     | `const char*` |        | empty    | any string |
+| OptName              | Since | Binding | Type       |  Units  |  Default  | Range  | Dir | Entity |
+| -------------------- | ----- | ------- | ---------- | ------- | --------- | ------ | --- | ------ |
+| `SRTO_STREAMID`      | 1.3.0 | pre     | `string`   |         | ""        | [512]  | RW  | GSD    |
+
+- A string that can be set on the socket prior to connecting. This stream ID
+will be able to be retrieved by the listener side from the socket that is
+returned from `srt_accept` and was connected by a socket with that set stream
+ID (so you usually use SET on the socket used for `srt_connect` and GET on the
+socket retrieved from `srt_accept`). This string can be used completely
+free-form, however it's highly recommended to follow the
+[SRT Access Control guidlines](AccessControl.md).
 
-- **[GET or SET]** - A string limited to 512 characters that can be set on the 
-socket prior to connecting. This stream ID will be able to be retrieved by the 
-listener side from the socket that is returned from `srt_accept` and was 
-connected by a socket with that set stream ID (so you usually use SET on the
-socket used for `srt_connect` and GET on the socket retrieved from `srt_accept`).
-This string can be used completely free-form, however it's highly recommended
-to follow the [SRT Access Control guidlines](AccessControl.md).
+- As this uses internally the `std::string` type, there are additional functions
+for it in the legacy/C++ API (udt.h): `srt::setstreamid` and
+`srt::getstreamid`.
 
-As this uses internally the `std::string` type, there are additional functions
-for it in the legacy/C++ API (udt.h): `UDT::setstreamid` and
-`UDT::getstreamid`. This option doesn't make sense in Rendezvous connection;
-the result might be that simply one side will override the value from the other
-side and it's the matter of luck which one would win
+- This option doesn't make sense in Rendezvous connection; the result might be
+that simply one side will override the value from the other side and it's the
+matter of luck which one would win
```

Comment:
```suggestion
- This option is not useful for a Rendezvous connection, since one side would 
override the value from the other side resulting in an arbitrary winner.
```